tests/bluetooth/bsim_bt/bsim_test_rpa_central/src/tester.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_bt_utils.h"
+#include "zephyr/bluetooth/addr.h"
+#include "zephyr/bluetooth/bluetooth.h"
+#include "zephyr/bluetooth/conn.h"
+#include "zephyr/toolchain/gcc.h"
+#include <host/hci_core.h>
+
+#include <stdint.h>
+#include <string.h>
+
+DEFINE_FLAG(flag_new_address);
+
+static struct bt_le_adv_param params;
+static struct bt_le_ext_adv_start_param start_params;
+static struct bt_le_ext_adv *adv;
```

Comment:
Suggest to move these to the `start_advertising` function as they are not used outside of it